off_psi4/confs2psi.py

```diff
@@ -92,12 +96,12 @@ def confs2psi(insdf, method, basis, spe=False, memory=None):
     ### For each molecule: for each conf, generate input
     for mol in ifs.GetOEMols():
         print(mol.GetTitle(), mol.NumConfs())
-        if not mol.GetTitle():
```

Comment:
@jmaat - I thought we do need titles on OEMols. Is there a reason this was taken out?